### PR TITLE
{vis}[GCCcore/11.3.0] cairo v1.17.4, GObject-Introspection v1.72.0, GLib v2.72.1, ... w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by jfgrimm**
depends on:

- [x] #15437 

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/b2290caeae228a2f1e662dc9ea011a96 for a full test report.

File: easybuild/easyconfigs/g/GLib/GLib-2.72.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+builddependencies = [
+    # Python is required for building against GLib, at least when
+    # gdbus-codegen or one of the other python scripts are used.
+    # Since Meson 0.50 and later are Python >=3.5 only we can't build
+    # Python specific versions of GLib that uses Python 2.x
+    # thus Python should not be a runtime dependency for GLib.
+    # Packages that use GLib should either have an explicit
+    # (build)dependency on Python or it will use the system version
+    # EasyBuild itself uses.
+    ('Python', '3.10.4', '-bare'),
```
[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyconfigs/g/GLib/GLib-2.72.1-GCCcore-11.3.0.eb:25]: And we explictly stop Lmod from swapping modules, which I guess is what is allowing this to build for others.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/7340e89346d59fe4e28e2df888d00a9d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/34f9f14f05961b2893edb6f68b952731 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0306u03a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 4 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/9798e33727831bce708e4c3dc1f3ed50 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 1 out of 4 (4 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/f6abad93ceea0ad868501bccdbc96d98 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 51 out of 51 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6978c448f653c7dd06125ecd6571ddff for a full test report.

**Comment by verdurin** [comment]
@SebastianAchilles any idea why your build failed? Could you retry?

**Comment by SebastianAchilles** [comment]
> @SebastianAchilles any idea why your build failed? Could you retry?

I have no idea what is causing the failed build in https://github.com/easybuilders/easybuild-easyconfigs/pull/15439#issuecomment-1118809488 This seems to happen only on my ubuntu 22.04 container. Right now I assume it is a problem with the container itself.

I will launch a VM with ubuntu 22.04 and create another test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
eb-u2204 - Linux Ubuntu 22.04, x86_64, Intel Xeon Processor (Skylake, IBRS) (skylake_avx512), Python 3.10.4
See https://gist.github.com/7dcc20dbc386279bd7ae97f9776e956a for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**